rust/datafusion/src/physical_plan/expressions.rs

```diff
@@ -1835,88 +1797,18 @@ mod tests {
         Ok(())
     }
 
-    #[test]
-    fn sum_contract() -> Result<()> {
```

Comment:
I removed these because the types are now selected from the signature, and no longer inferred by `AggregateExpr`.

Reply:
makes sense